operator/pkg/central/extensions/reconcile_pvc.go

```diff
@@ -67,21 +68,33 @@ func convertDBPersistenceToPersistence(p *platform.DBPersistence) *platform.Pers
 	}
 }
 
-func getPersistenceByClaimName(central *platform.Central, claim string) *platform.Persistence {
-	switch claim {
-	case DefaultCentralPVCName:
-		return central.Spec.Central.GetPersistence()
-	case DefaultCentralDBPVCName:
-		return convertDBPersistenceToPersistence(central.Spec.Central.DB.GetPersistence())
+func getPersistenceByTarget(central *platform.Central, target PVCTarget) *platform.Persistence {
```

Comment:
Can you add a small doc string what this function is used for? I found it hard to understand, looking into the caller also did not really helped me.